src/python/gudhi/wasserstein/wasserstein.py

```diff
@@ -173,9 +173,40 @@ def wasserstein_distance(X, Y, matching=False, order=2., internal_p=2., enable_a
         return ep.concatenate(dists).norms.lp(order).raw
         # We can also concatenate the 3 vectors to compute just one norm.
 
-    # Comptuation of the otcost using the ot.emd2 library.
+    # Comptuation of the otcost using the ot.emd2 library, and matching/transport plan using ot.emd.
```

Comment:
```suggestion
    # Computation of the otcost using the ot.emd2 library, and matching/transport plan using ot.emd.
```
might as well fix the typo at the same time